lib/node_modules/@stdlib/stats/base/dists/planck/mode/test/fixtures/python/runner.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+#
+# @license Apache-2.0
+#
+# Copyright (c) 2024 The Stdlib Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Generate fixtures."""
+
+import os
+import json
+import numpy as np
+
+# Get the file path:
+FILE = os.path.realpath(__file__)
+
+# Extract the directory in which this file resides:
+DIR = os.path.dirname(FILE)
+
+
+def gen(lam, name):
+    """
+    Generate fixture data and write to file.
+
+    # Arguments
+
+    * `lam`: shape parameter.
+    * `name::str`: output filename.
+
+    # Examples
+
+    ```python
+    python> lam = np.random.rand(1000) * 20
+    python> gen(lam, "data.json")
+    ```
+    """
+    # Compute standard deviation values:
+    z = np.zeros(lam.shape)
```

Comment:
I know this is not the right way to proceed. But there is no attribute named `mode` in scipy.

Reply:
In this case, it would be better to not have any test fixtures but to change the tests to simply compare the output to the correct value. Should also include a comment with a link to a source for the value of the mode of the Planck distribution.